core/container/src/main/java/org/wildfly/swarm/container/config/ConfigViewFactory.java

```diff
@@ -104,7 +104,7 @@ public ConfigViewFactory load(String profileName) {
     public void load(String profileName, URL url) throws IOException {
         if (url.getPath().endsWith(".properties")) {
             loadProperties(profileName, url);
-        } else if (url.getPath().endsWith(".yml")) {
+        } else if (url.getPath().endsWith(".yml") || url.getPath().endsWith(".yaml")) {
             loadYaml(profileName, url);
         }
     }
```

Comment:
I don't think here, but a warning somewhere would be useful.

Problem is `-Pfoo` can be valid with `project-stages.yaml` defining a `foo` stage, so we can't explicitly care about the file names.

I'll follow with another PR if you don't mind adding a JIRA about issuing a warning for `an activated profile without a matching profile defined`.